djangocms_versioning_filer/plugins/video/models.py

```diff
@@ -6,15 +6,19 @@
 from cms.models import CMSPlugin
 
 from djangocms_attributes_field.fields import AttributesField
-from djangocms_video.models import VideoTrack, get_extensions, get_templates
+from djangocms_video.models import VideoTrack, get_templates
 
 from djangocms_versioning_filer.fields import (
     FileGrouperField,
     ImageGrouperField,
 )
 
 
-ALLOWED_EXTENSIONS = get_extensions()
+try:
```

Comment:
@jonathan-s Hate to be a pain, can we add a comment detailing which versions of djangocms_video forced this change, if we detail version numbers one day in the future they will be redundant and this can be cleaned out.